elem/elem.gen.go

```diff
@@ -788,55 +759,56 @@ func Span(markup ...vecty.MarkupOrChild) *vecty.HTML {
 	return vecty.Tag("span", markup...)
 }
 
-// Strong gives text strong importance, and is typically displayed in bold.
+// Strong indicates that its contents have strong importance, seriousness, or
+// urgency. Browsers typically render the contents in bold type.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong
 func Strong(markup ...vecty.MarkupOrChild) *vecty.HTML {
 	return vecty.Tag("strong", markup...)
 }
 
-// Style contains style information for a document, or part of a document. By
-// default, the style instructions written inside that element are expected to
-// be CSS.
+// Style contains style information for a document, or part of a document.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style
 func Style(markup ...vecty.MarkupOrChild) *vecty.HTML {
 	return vecty.Tag("style", markup...)
 }
 
-// Subscript defines a span of text that should be displayed, for typographic
-// reasons, lower, and often smaller, than the main span of text.
+// Subscript specifies inline text which should be displayed as subscript for
+// solely typographical reasons.
 //
 // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub
 func Subscript(markup ...vecty.MarkupOrChild) *vecty.HTML {
 	return vecty.Tag("sub", markup...)
 }
 
-// Summary is used as a summary, caption, or legend for the content of a
-// <details> element.
+// Summary element specifies a summary, caption, or legend for a <details>
```

Comment:
We'll need to fix this to remove the word `element` or make it a proper sentence somehow.